crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -560,7 +560,15 @@ pub fn derive_event(input: TokenStream) -> TokenStream {
     component::derive_event(input)
 }
 
-/// Implement the `EntityEvent` trait.
+/// Cheat sheet for derive syntax,
```

Comment:
I think the main thing someone would be after here is a simple example they can copy paste, so it would be useful to include `derive(EntityEvent)` in these snippets as well as the `entity_events` attributes